src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -156,21 +154,14 @@ public void sendBuildMessage(StepParameters stepParameters) {
 
         String webhookUrl = stepParameters.getWebhookUrl();
         try {
-            if (webhookUrl != null) {
-                //               listener.getLogger().println(String.format("Notifying webhook '%s'", webhookUrl));
-                HttpWorker worker = new HttpWorker(run.getEnvironment(listener).expand(webhookUrl), gson.toJson(card), 30000, 3, listener.getLogger());
+            WebhookJobProperty property = (WebhookJobProperty) run.getParent().getProperty(WebhookJobProperty.class);
```

Comment:
in fact webhookurl was never ``null`` because as mentioned below, ``null`` value was converted into empty string.

Never the less, this if-else block is now shorten